Engine/src/Engine/SceneBuilder/Scene.h

```diff
@@ -4,6 +4,7 @@
 #include "Components.h"
 #include <Engine/Utilities/DeltaTime.h>
 #include "Engine/Core.h"
+#include <Engine/ResourceManagement/ShaderGenerator.h>
```

Comment:
This can be forward declared

Reply:
What do you mean by this? Scene.h needs to access it because it has a singleton shaderNorms object. In ShaderNorms.h now.

Reply:
Done